seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestConstant.java

```diff
@@ -27,6 +27,8 @@ public class RestConstant {
 
     public static final String IS_STOP_WITH_SAVE_POINT = "isStopWithSavePoint";
 
+    public static final String ENV_CONFIG_STYLE = "configStyle";
```

Comment:
```suggestion
    public static final String ENV_CONFIG_FORMAT = "format";
```
I think use the format more clear.